containers/datalab/Dockerfile.in

```diff
@@ -50,7 +50,7 @@ RUN pip install -U tornado==4.3 \
                    argparse==1.2.1 \
                    mock==1.2.0 \
                    requests==2.9.1 \
-                   oauth2client==1.5.2 \
+                   oauth2client==2.0.2 \
```

Comment:
@dhermes This is `oauth2client == 2.0.2` for consistency with #804. Now that the requirement is compatible with `gcloud-python` (yay!), that's good enough for the purposes of this pull request.